src/app/core/components/redeem-gift/redeem-gift.component.html

```diff
@@ -1,15 +1,19 @@
 <!-- @format -->
-<div class="panel-title">Redeem Gift</div>
+<div class="panel-title">Redeem Storage Code</div>
 <p>
-  If you've been given a gift code, you can redeem it for free storage below.
+  If you have received a storage code, you can redeem it for free storage below.
+  <a
+    href="https://permanent.zohodesk.com/portal/en/kb/articles/redeem-a-storage-code"
```

Comment:
Message for QA: Do we want to open this in a new tab since this link is in a modal?